src/main/java/com/weavers/duqhan/business/impl/PaymentServiceImpl.java

```diff
@@ -0,0 +1,435 @@
+import com.weavers.duqhan.business.UsersService;
+import com.weavers.duqhan.dao.CartDao;
+import com.weavers.duqhan.dao.OrderDetailsDao;
+import com.weavers.duqhan.dao.PaymentDetailDao;
+import com.weavers.duqhan.dao.ProductSizeColorMapDao;
+import com.weavers.duqhan.dao.ShipmentTableDao;
+import com.weavers.duqhan.dao.UserAddressDao;
+import com.weavers.duqhan.dao.UsersDao;
+import com.weavers.duqhan.domain.Cart;
+import com.weavers.duqhan.domain.OrderDetails;
+import com.weavers.duqhan.domain.PaymentDetail;
+import com.weavers.duqhan.domain.ProductSizeColorMap;
+import com.weavers.duqhan.domain.ShipmentTable;
+import com.weavers.duqhan.domain.UserAddress;
+import com.weavers.duqhan.dto.AddressDto;
+import com.weavers.duqhan.dto.CartBean;
+import com.weavers.duqhan.util.GenerateAccessToken;
+import com.weavers.duqhan.dto.ProductBean;
+import com.weavers.duqhan.util.CurrencyConverter;
+import com.weavers.duqhan.util.PayPalConstants;
+import com.weavers.duqhan.util.StatusConstants;
+import java.io.InputStream;
+import java.text.DecimalFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ *
+ * @author Android-3
+ */
+public class PaymentServiceImpl implements PaymentService {
+
+    @Autowired
+    UsersDao usersDao;
+    @Autowired
+    PaymentDetailDao paymentDetailDao;
+    @Autowired
+    OrderDetailsDao orderDetailsDao;
+    @Autowired
+    ProductSizeColorMapDao productSizeColorMapDao;
+    @Autowired
+    UserAddressDao userAddressDao;
+    @Autowired
+    CartDao cartDao;
+    @Autowired
+    ShippingService shippingService;
+    @Autowired
+    ShipmentTableDao shipmentTableDao;
+    @Autowired
+    UsersService usersService;
+    @Autowired
+    NotificationService notificationService;
+    SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd");
+
+    private final Logger logger = Logger.getLogger(PaymentServiceImpl.class);
+
+    @Override
+    public String[] transactionRequest(HttpServletRequest request, HttpServletResponse response, CartBean cartBean, Double shippingCost, List<Shipment> shipments) {
+        String[] strArray = new String[2];
+        String returnUrl = null;
+        String payKey = null;
+        InputStream is;
+        String paypalToken = null;
+        DecimalFormat df2 = new DecimalFormat(".##");
+        List<ProductBean> productBeans = cartBean.getProducts();
+        is = this.getClass().getResourceAsStream("/sdk_config.properties");
+        try {
+            PayPalResource.initConfig(is);
+        } catch (PayPalRESTException ex) {
+            logger.error("(==E==)PayPalRESTException for PayPalResource.initConfig ", ex);

```

Comment:
if there is an error do you want to continue further execution of the code? or should you return an error saying the transaction cannot be done successfully?